### ISSUE TITLE
Regression in CommaRule from 0.9.1 introduced in 0.9.2

### ISSUE THREAD

**Issue by andymatuschak** [realm/SwiftLint#683]
Hello! With swiftlint 0.9.1, the following source listing correctly produces a `CommaRule` violation. As of swiftlint 0.9.2, it no longer produces the violation.

```swift
func plus(first: Int, second: Int) -> Int {
	return first + second
}

let result = plus(
	first: 3 , // <--- this should produce an error
	second: 4
)
```

cc @norio-nomura

**Comment by jpsim** [comment]
good to know! thanks for reporting.

**Comment by andymatuschak** [comment]
The regression was introduced in 6f17c93bd621eb6aac1c2f5f837cfbd813a7d8b0.

**Comment by norio-nomura** [comment]
Thanks for reporting! :pray:
I pushed fix #685

**Comment by andymatuschak** [comment]
Woo! Thanks. :D
